master/buildbot/test/unit/test_worker_ec2.py

```diff
@@ -138,7 +138,7 @@ def test_start_vpc_instance(self):
         bs = ec2.EC2LatentWorker('bot1', 'sekrit', 'm1.large',
                                  identifier='publickey',
                                  secret_identifier='privatekey',
-                                 keypair_name="test_key",
+                                 keypair_name="latent_buildbot_slave",
```

Comment:
The test fixture sets up `latent_buildbot_slave`, I can change the test fixture to use `latent_buildbot_worker` and update the whole test suite , but I would also have to change the defaults in [ec2.py#L76](https://github.com/buildbot/buildbot/blob/c3969c43b861f00abf7653c02b80cd0e8b8741a6/master/buildbot/worker/ec2.py#L76)

Want me to change it for both, or is there some backwards compatibility that needs to be maintained? for `_slave`?